oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -609,26 +609,70 @@ private APIGatewayProxyResponseEvent handleAuthJourney(
                     authenticationRequest.getClientID().getValue(),
                     user);
         }
-        var session = existingSession.orElseGet(sessionService::generateSession);
-        attachSessionIdToLogs(session);
 
+        Session session;
+        OrchSessionItem orchSession;
+        var maxAgeParam = getMaxAge(authenticationRequest);
```

Comment:
Could this be in the `else`? As there's no point calculating it if the session is empty

Reply:
Ooh yeah you're right

Reply:
Have moved this and the timeNow variables into the else block